README.md

```diff
@@ -1 +1,19 @@
-# TestRepo
\ No newline at end of file
+# Selenium first steps
+
+Script(Java language) for Selenium WebDriver that opens Facebook.com starting page.
+
+## Installation
+
+1.Install Java
+2.Install Eclipse IDE
+3.Install Selenium WebDriver
+4.Install Chrome WebDriver
+5.Run script in Eclipse IDE
```

Comment:
Your list is displayed as:
![screen shot 2019-02-09 at 3 34 04 pm](https://user-images.githubusercontent.com/17481353/52527593-41a0bd00-2c80-11e9-9eb7-3103a54c1480.png)
It should be a numbered list where each element is on the new line